File: S10-animaciones/CL02-AnimacionesAutomaticas/app/src/main/java/dev/miguelcampos/cl02_animacionesautomaticas/MainActivity.java
package dev.miguelcampos.cl02_animacionesautomaticas;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    TextView tvInfo;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        tvInfo = findViewById(R.id.textViewInformacion);
    }

    public void cambiarVisibilidad(View view) {
        if(tvInfo.getVisibility() == View.VISIBLE) {
            tvInfo.setVisibility(View.GONE);
        } else {
            tvInfo.setVisibility(View.VISIBLE);
        }
    }
}

File: S03-conceptos-basicos/CL07Checkboxes/app/src/main/java/com/androidavanzado/cl07checkboxes/MainActivity.kt
package com.androidavanzado.cl07checkboxes

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.CheckBox
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun clickOnHobby(view: View) {
        val checkBox: CheckBox = view as CheckBox
        val checked: Boolean = checkBox.isChecked

        when(view.id) {
            R.id.checkBoxDeporte -> {
                if(checked) {
                    Toast.makeText(this, "Se ha marcado Deporte", Toast.LENGTH_LONG).show()
                } else {
                    Toast.makeText(this, "Se ha Desmarcado Deporte", Toast.LENGTH_LONG).show()
                }
            }
            R.id.checkBoxVideojuegos -> {
                if(checked) {
                    Toast.makeText(this, "Se ha marcado Videojuegos", Toast.LENGTH_LONG)
                        .show()
                } else {
                    Toast.makeText(this, "Se ha Desmarcado Videojuegos", Toast.LENGTH_LONG)
                        .show()
                }
            }
        }
    }
}
